Attachments/adding-attachments.md

```diff
@@ -0,0 +1,251 @@
+## How to upload and attach files from a third party applications
+
+The basic flow of uploading files in Lime Field looks like this:
+
+1. Create a `File` entity
+2. Upload file content to that entity's endpoint
```

Comment:
Don't think we need to mention endpoint here since we don't do that in point 1 or 3. Just say somethink along the line of `Upload the file's content`.